screen-search/src/main/java/com/ivy/search/SearchViewModel.kt

```diff
@@ -60,6 +65,9 @@ class SearchViewModel @Inject constructor(
     private fun search(query: String) {
         val normalizedQuery = query.lowercase().trim()
 
+
+        sharedPrefs.putString(SharedPrefs.SEARCH_SCREEN_REMEMBER, normalizedQuery);
```

Comment:
Can we use the Android DataStore instead? Ivy Wallet is migrating to it and also it's main-safe

Reply:
We should also persist the search query in a separate coroutine so it doesn't block the search refresh logic

Reply:
Let me work on these over the weekend